rehack_tests/js/js_of_ocaml.cma.php/Js_of_ocaml__Lib_version.php

```diff
@@ -31,7 +31,7 @@ private static function load($joo_global_object) {
     $runtime = $joo_global_object->jsoo_runtime;
     $string = $runtime["caml_new_string"];
     $s = $string("3.3.0");
-    $git_version = $string("35378b53");
```

Comment:
Not sure what this change is. Will revert

Reply:
either way. Doesn't really matter.